lib/pbench/server/api/resources/query_apis/datasets/dataset_mappings.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
             ),
         )
 
-    def _get(self, index_key: AnyStr, request: Request) -> Dict[AnyStr, Any]:
+    def _get(self, dataset_view: AnyStr, request: Request) -> Dict[AnyStr, Any]:
         """
         Return mapping properties of the document specified by the index key
         in the URI (supported index keys are defined in RunIdBase.ES_INTERNAL_INDEX_NAMES).
```

Comment:
I will just use the dataset view

Reply:
I think we probably ought to be consistent in naming from the abstract declaration in the superclass to here, and the generic isn't specific to your view parameter, datasets or otherwise. These are generically the JSON parameters to the call, although in the specific case of `_get` that's only the accumulated URI parameters as we don't support a JSON payload.

The abstract just uses "json_data" so let's stick to that. The docstring can point out the meaning of the specific keys we expect (in this context) to find withing `json_data`.